op-service/clock/clock.go

```diff
@@ -13,6 +14,10 @@ type Clock interface {
 	// Now provides the current local time. Equivalent to time.Now
 	Now() time.Time
 
+	// Add returns the time t+d. Equivalent to time.Now().Add(d)
+	// If d is negative, the duration is safely subtracted from t.
+	Add(d time.Duration) time.Time
```

Comment:
I don't think `Add` is the right name for this as it suggests that the time in the clock is being changed.  I'd suggest `NowPlus`.